### ISSUE TITLE
ScrollView: Crashes if ShowXXXScrollIndicator is false

### ISSUE THREAD

**Issue by tig** [migueldeicaza/gui.cs#597]
To repro change `Scrolling` Scenario as follows:

```csharp
			scrollView.ColorScheme = Colors.TopLevel;
			scrollView.ContentSize = new Size (200, 100);
			//ContentOffset = new Point (0, 0),
			scrollView.ShowVerticalScrollIndicator = false;
			scrollView.ShowHorizontalScrollIndicator = false;
```

**Comment by BDisp** [comment]
In the `ScrollView `should be:

```
		public override void Redraw(Rect region)
		{
			Driver.SetAttribute (Colors.Menu.Normal);

```

**Comment by BDisp** [comment]
Fixes here #599.
